frontend/integration-tests/protractor.conf.ts

```diff
@@ -15,6 +15,7 @@ export const testName = `test-${Math.random().toString(36).replace(/[^a-z]+/g, '
 
 const htmlReporter = new HtmlScreenshotReporter({dest: './gui_test_screenshots', inlineImages: true, captureOnlyFailedSpecs: true, filename: 'test-gui-report.html'});
 const browserLogs: logging.Entry[] = [];
+const junitReporter = new JUnitXmlReporter({savePath: 'xunit_results.xml', consolidateAll: true});
```

Comment:
I'd probably put this in the (increasingly inaccurately-named) gui_test_screenshots folder since it has other test results like test-gui-report.html and bundle analysis.

We could look at renaming it, but I'd need to check if it impacts on our CI.